linera-core/src/unit_tests/worker_tests.rs

```diff
@@ -2924,7 +2939,7 @@ where
         &worker,
         HashedValue::new_confirmed(
             BlockExecutionOutcome {
-                messages: Vec::new(),
+                messages: vec![],
```

Comment:
I am pretty neutral in those matters. For me `vec![]` and `Vec::new()` is all the same.
But in at line 1296 you go one way and at line 2927, you go the other way.